appengine/pond/duck/js/duck.js

```diff
@@ -125,53 +125,28 @@ Pond.Duck.init = function () {
       'damage,health,loc_x,getX,loc_y,getY,');
 };
 
+
 /**
- * Load default players to pond game.
+ * Load default players for the the pond game.
+ * @param {?string} currentPlayerName The name of the currently loaded duck or null.
  */
-Pond.Duck.loadDefaultPlayers = function () {
-  var players = [
-    {
-      start: new Blockly.utils.Coordinate(20, 80),
-      damage: 0,
-      name: 'Pond_myName',
-      code: null
-    },
-    {
-      start: new Blockly.utils.Coordinate(80, 80),
-      damage: 0,
-      name: 'Pond_rookName',
-      code: 'playerRook'
-    },
-    {
-      start: new Blockly.utils.Coordinate(20, 20),
-      damage: 0,
-      name: 'Pond_counterName',
-      code: 'playerCounter'
-    },
-    {
-      start: new Blockly.utils.Coordinate(80, 20),
-      damage: 0,
-      name: 'Pond_sniperName',
-      code: 'playerSniper'
-    }
-  ];
-  Pond.Duck.loadPlayers(players);
+Pond.Duck.loadDefaultAvatars = function (currentPlayerName) {
```

Comment:
currentPlayerName should be optional (with opt_) and JsDoc would need update as well.